bigtable/examples/run_examples_utils.sh

```diff
@@ -19,6 +19,47 @@ readonly CBT_CMD="${CBT:-${GOPATH}/bin/cbt}"
 readonly CBT_EMULATOR_CMD="${CBT_EMULATOR:-${GOPATH}/bin/emulator}"
 readonly CBT_INSTANCE_ADMIN_EMULATOR_CMD="../tests/instance_admin_emulator"
 
+function cleanup_instance {
+  local project=$1
```

Comment:
This line will exit if `$1 `is unspecified given `set -eu` at the top of the file (just tested to verify). This should be `"${1:-}"` to provide it a default. Then, it should immediately test to see if it's empty, and if so, show an error and return so it's clear why it's returning.

Reply:
Or I could do just like in the other functions and just don't ever pass empty values.

Reply:
As you prefer.